static/cdn/maintenance.html

```diff
@@ -0,0 +1,208 @@
VEQ4BIyczNSMVBSEiJjURNDYzITIWFREUBiMnMzUjFQUhIiY1ETQ2MyEyFhURDgEjJzM1IxUDISImNRE0NjMhMhYVERQGIyczNSMVAU7+9xcbGxcBCRMfBRoT3q2tAon+9hIZGRIBChIfHxLYra0CiP73Ex8bFwEJEx8FGhPera3S/vYSGRkSAQoSHx8S2K2tNB8TAQoSGRoY/v0TH12trV0fEwEKEhkaGP79Ex9dra1dGxcBChIfGhf+9hMfXa2tAVQZEgEKFxsbF/72Ehldra0AAAADAAD/wAOvA8AANQBCAE8AAAEmBgcOAycmBgcOARURLgEjIgYVFBYzMjY3PAE1ERY+AjcRLgEjIgYVFBYzMjY1ETQmJwEiJjU0NjMyFhUUBiMlIiY1NDYzMhYVFAYjA40PIwoudoaPRwobCgUJFCoZTHFxTEtsBT99d24wFCoZTHFxTEtxEhD9MCQ0NCQjNDQjAjYkNDQkIzQ0IwPABQgKOE8qARcFBwUKGwr+QwoKcUtMcWhHBQQFAiEMBB43Jv5sCgpxTExwcEwCvQ8bBfxlNCQjNDQjJDReNSMjNTUjIzUAAAAEAFH/9wTgA4IACQBLAFkAZwAAJTgBMRQyJxQ0MQEuAyMiBgcuASMiDgIHDgEVERQWFxY2Nz4BMhYXMBYxMDIxFjI3MDIxMDY1MT4BMhYXHgEzMjY3PgE1ETYmJwERPgEyFhcRLgEjIgYHIS4BIyIGBxE+ATIWFxECiAUFAlEgSExQKEaNNzyQRihRTUgeCgoRCg8cCjJ7gHsyBgcKEgUHBzJ7f3syBRIKBQoFDwwKBwr73DBydXIvMnM3PG4yA9Qyczc3czIwcnVyLwUFBQUFAxIZKBsPLygoLw8bKBkKFAr9Pg8WCgUCCigoKCgHBQUCBSgoKCgFCQIFBRsPAsIKFAr9cwJLIyMjI/21GRwcGRkcHBkCSyMjIyP9tQAAAAABAFP/zQQAA6AAQgAAAREUBiMhIiY/AS4BJzgBMS4BDgEHDgEeARceAT4BNz4BFx4BBw4DBwYuAicuAjY3PgIWFzAWFx4BFzc2FhUEABgO/pMdEBOADhoFOYB6bSYmGRZFNzd9emwmE0odHQ4YHUdQWC4tX15aKFRrJSQ5N6G3vVQIBQokGHoTLQOA/oATEzMThg8TBSkbFUU3N4GAcikpGxZHOh0OGBNKHCxFNCEHCgIWKB08rMHHVlRpIiU6AQUKHBR6ExYdAAIAAP/ABAADwAAUACsAAAEiDgIVFB4CMzI+AjU0LgIjCQEOASMiJi8BJjQ3NjIfAQE2MhcWBgcCAGu7ilBQirtra7uKUFCKu2sBHf7NCx0LCyULmRYWFjwbZwEAFkIWEAIWA8BQirtra7uKUFCKu2tru4pQ/mb+xgsLEAahFkIWFRVnAQAWFhZCFgAACABM/74DtAPAAA4AbABzAIAAiwCWAKEArAAACQEuASceARcBHgEXLgEnFzwBJzwBJzwBNS4BJwEuAScwIjUiMCM0IjEmIiM4ATEmIiMqASMwJiMqASMhIgYVERQWFx4BMzI2NRE0NjsBFRQWOwERFAYjISImJy4BKwEiBhceATMhMjY1ETwBNSciJj0BFyMBMzIWHQEUBisBFSM1FzI2PQE0JisBFTM3MzIWHQEUBisBNRcyNj0BNCYrARUzNzMVIxUzFSMVIzUDqP7NAgQCAgQCATMDBQEBBQMMAQECBAT+zQEFAgEBAQIBAQEBAQEBAQEBAQECAf7wPlkHBQYOCRAYKR3oWT6cKR7+LBYjCAQRCwYUGAgQTC8B1D9Y7B0qqmP9hF8qMzYqHj5bEBMTERwdmlY8QkM8VVccIiIcGRm8snRpaT4CggEyAgMCAgMC/s4DBwQDCAMYAQEBAQIBAQEBBAcDATICAwIBAQEBAVk+/uIIDgYFBxgQAR4dKpw+Wf4YHSoYEwoOIhIqNVg/AhEBAgEkKR1jqf75KiQBJylA328QDQEODztvPjABMEDfqB0bARsdcag2JjNQ3wAAAAEAAAABAACjOEtzXw889QALBAAAAAAA0d/e2gAAAADR397aAAD/vgTgA8AAAAAIAAIAAAAAAAAAAQAAA8D/wAAABT8AAAAABOAAAQAAAAAAAAAAAAAAAAAAABIEAAAAAAAAAAAAAAACAAAABAAA2AJ6AHICegCUBCoAYQQAAHIEAAB0BAcADwQAAAAE6AATA68AAAU/AFEEAABTBAAAAAQAAEwAAAAAAAoAFAAeAFwAhgCwAPIBDgE8AdoCNgK0AyYDtgQaBGAFQAAAAAEAAAASAK0ACAAAAAAAAgAAAAAAAAAAAAAAAAAAAAAAAAAOAK4AAQAAAAAAAQANAAAAAQAAAAAAAgAHAJYAAQAAAAAAAwANAEgAAQAAAAAABAANAKsAAQAAAAAABQALACcAAQAAAAAABgANAG8AAQAAAAAACgAaANIAAwABBAkAAQAaAA0AAwABBAkAAgAOAJ0AAwABBAkAAwAaAFUAAwABBAkABAAaALgAAwABBAkABQAWADIAAwABBAkABgAaAHwAAwABBAkACgA0AOxzY3JhdGNoLWljb25zAHMAYwByAGEAdABjAGgALQBpAGMAbwBuAHNWZXJzaW9uIDEuMABWAGUAcgBzAGkAbwBuACAAMQAuADBzY3JhdGNoLWljb25zAHMAYwByAGEAdABjAGgALQBpAGMAbwBuAHNzY3JhdGNoLWljb25zAHMAYwByAGEAdABjAGgALQBpAGMAbwBuAHNSZWd1bGFyAFIAZQBnAHUAbABhAHJzY3JhdGNoLWljb25zAHMAYwByAGEAdABjAGgALQBpAGMAbwBuAHNGb250IGdlbmVyYXRlZCBieSBJY29Nb29uLgBGAG8AbgB0ACAAZwBlAG4AZQByAGEAdABlAGQAIABiAHkAIABJAGMAbwBNAG8AbwBuAC4AAAADAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA');
```

Comment:
I think it's a great idea to keep these here. If we used webpack to build these, then we could use data-uri's for static assets, but store the files like normal in here.

Reply:
I agree! I'm setting up a separate issue to have this be part of the webpack processing and then run through the configure-fastly.js  as part of the deploy step. See: #2125